packages/react/src/components/Authenticator/Authenticator.tsx

```diff
@@ -20,6 +20,22 @@ export type AuthenticatorProps = AuthenticatorMachineOptions &
   RouterProps &
   ComponentsProviderProps;
 
+// Helper component that sends init event to the parent provider
+function InitMachine({ children, ...machineProps }) {
```

Comment:
Nit: replace `machineProps` with `data` to allow object shorthand notation on ln 30